src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1670,6 +1670,18 @@ GlobeControls.prototype.pixelsToMeters = function pixelsToMeters(pixels, pixelPi
 };
 
 /**
+ * To convert the projection in degree WGS84 on the globe of a number of pixels of screen
+ * @param      {number} pixels count pixels to project
+ * @param      {number} pixelPitch Screen pixel pitch, in millimeters (default = 0.28 mm / standard pixel size of 0.28 millimeters as defined by the OGC)
+ * @return     {number} projection in degree on globe
+ */
+GlobeControls.prototype.pixelsToDegrees = function pixelsToDegrees(pixels, pixelPitch = 0.28) {
+    const chord = this.pixelsToMeters(pixels, pixelPitch);
+    const rayon = ellipsoidSizes().x;
+    return 2 * Math.asin(chord / (2 * rayon)) * 180 / Math.PI;
```

Comment:
Please use `THREE.Math.radToDeg`: easier to read than `180 / Math.PI`.
And use english var names -> `rayon` -> `radius`...
Btw, I can't see how  this formula can be correct on an ellipsoid with different radius values on each axes?

Reply:
Last but not least: this method doens't belong to `*Controls`.
`GlobeView` is a better place or a new `GlobeUtils` module if there are other utils functions like this one.

Reply:
There are several functions to move, I'll do a special PR